cpp/include/utilities/cython.hpp

```diff
@@ -246,6 +301,29 @@ void call_sssp(raft::handle_t const& handle,
                vertex_t* predecessors,
                const vertex_t source_vertex);
 
+// wrapper for shuffling:
+//
+template <typename vertex_t, typename edge_t, typename weight_t>
+major_minor_weights_t<vertex_t, weight_t> call_shuffle(
+  raft::handle_t const& handle,
+  vertex_t* edgelist_major_vertices /* [IN] */,  // make_zip_iterator does not accept const !
+  vertex_t* edgelist_minor_vertices /* [IN] */,
+  weight_t* edegelist_weights,
```

Comment:
spelling error here `edege` should be `edge`

Reply:
Fixed.